b2share/version.py

```diff
@@ -20,4 +20,4 @@
 """Version number."""
 
 
-__version__ = "2.0.0.rc3"  # FIXME: generate the version number
+__version__ = "2.0.0.rc4"  # FIXME: generate the version number
```

Comment:
@jirikuncar Thanks. I agree that we should follow python standard for release numbering. I'll check with @emanueldima what is the reason for this custom numbering when he is back, which means that will keep this version for this RC.